deployment/terraform.tfvars.example

```diff
@@ -18,6 +19,7 @@ web_environment_variables = {
   BASIC_AUTH_PASSWORD = "secret"
 }
 scrapy_image_tag = "sha-123"
+scrapy_replica_count = 3
```

Comment:
Feels a little like overkil for scrapy, but I suppose it doesn't cause any harm.